index.html

```diff
@@ -383,15 +393,22 @@
         }
     }
     let rounds = [];
+    const removed_districts = gState.filters
+        .map(x => /^district-d(\d+)$/.exec(x)).filter(m => m != null).map(m => m[1])
+        .reduce((obj, x) => {obj.set(x, true); return obj;}, new Map())
```

Comment:
One other comment:  I think this (and the equivalent 3 lines later) can be simplified by just passing the array to the `Set()` constructor rather than using `reduce` to construct a `Map`.  It's could also probably be simplified by just doing a single `map` that replaces any `"district-d"` at the start of the string with `""`.

Reply:
Thanks. Not exactly sure about removing the prefix though.